metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -58,13 +60,19 @@ class AzureADConfig(ConfigModel):
     users_pattern: AllowDenyPattern = AllowDenyPattern.allow_all()
     groups_pattern: AllowDenyPattern = AllowDenyPattern.allow_all()
 
+    filteredTracking: bool = True
```

Comment:
nitpick: in python variable naming, we tend to use "_" underscore to separate words instead of camelCase! Can you adjust these variable names?

Reply:
On this one, can we rename it to `report_filtered`